### PR TITLE
fix: Integrations quality updates

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Fix issues uncovered when reviewing integrations list and form.

- YouTube CSP
- Text content and formatting
- Margins
- Update old integration icons
- Fix headers in dark theme

### CODE REVIEW

File: src/lib/create-config.ts [hunk 1]
```diff
@@ -326,13 +329,20 @@ const parseCspEnvironmentVariables = (): ICspDomainConfig => {
     const scriptSrc = process.env.CSP_ALLOWED_SCRIPT?.split(',') || [];
     const imgSrc = process.env.CSP_ALLOWED_IMG?.split(',') || [];
     const connectSrc = process.env.CSP_ALLOWED_CONNECT?.split(',') || [];
+    const mediaSrc = process.env.CSP_ALLOWED_MEDIA?.split(',') || [];
+    const objectSrc = process.env.CSP_ALLOWED_OBJECT?.split(',') || [];
+    const frameSrc = process.env.CSP_ALLOWED_FRAME?.split(',') || [];
```
Comment (Tymek) [inline src/lib/create-config.ts:334]: Up until now this was derived from "default CSP". If someone depended on default CSP to embed frames/objects/media this has a potential to break things.

Comment (ivarconr) [reply src/lib/create-config.ts:334]: It makes sense to support all, but I really struggle to see the value of custom values here.

**Review by ivarconr — APPROVED** [review]

File: frontend/src/component/integrations/IntegrationList/IntegrationCard/IntegrationCardMenu/IntegrationCardMenu.tsx [hunk 2]
```diff
@@ -21,6 +21,8 @@ import type { AddonSchema } from 'openapi';
 import useAddons from 'hooks/api/getters/useAddons/useAddons';
 import useToast from 'hooks/useToast';
 import { formatUnknownError } from 'utils/formatUnknownError';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import { event } from 'cypress/types/jquery';
```
Comment (thomasheartman) [inline frontend/src/component/integrations/IntegrationList/IntegrationCard/IntegrationCardMenu/IntegrationCardMenu.tsx:25]:
```suggestion
```

File: website/docs/reference/deploy/configuring-unleash.md [hunk 3]
```diff
@@ -102,13 +102,16 @@ unleash.start(unleashOptions);
...
-- **additionalCspAllowedDomains** (CspAllowedDomains) - use this when you want to enable security headers but have additional domains you need to allow traffic to
-  - You can set the environment variable CSP_ALLOWED_DEFAULT to allow new defaultSrc (comma separated list)
-  - You can set the environment variable CSP_ALLOWED_FONT to allow new fontSrc (comma separated list)
-  - You can set the environment variable CSP_ALLOWED_STYLE to allow new styleSrc (comma separated list)
-  - You can set the environment variable CSP_ALLOWED_SCRIPT to allow new scriptSrc (comma separated list)
-  - You can set the environment variable CSP_ALLOWED_IMG to allow new imgSrc (comma separated list)
-  - You can set the environment variable CSP_ALLOWED_CONNECT to allow new connectSrc (comma separated list)
+- **additionalCspAllowedDomains** (CspAllowedDomains) - use this when you want to enable security headers but have additional domains you need to allow traffic to you can set the following environment variables:
```
Comment (thomasheartman) [inline website/docs/reference/deploy/configuring-unleash.md:105]:
```suggestion
- **additionalCspAllowedDomains** (CspAllowedDomains) - use this when you want to enable security headers but have additional domains you need to allow traffic to. You can set the following environment variables:
```

**Review by thomasheartman — APPROVED** [review]

**Pull request merged**